scl_prefixes.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Should this file live in gem2rpm directory?

Reply:
In a future patch I'm also going to use this in a `get-gemfile-deps` script which converts `Gemfile` + `bundler.d/*.rb` into a usable format to update all the sections in `foreman.spec`. I'm fine with it living in `gem2rpm`, but it's not strictly a `gem2rpm` thing anymore.